### ISSUE TITLE
Accessibility: [Screen Reader ]: The list of <ul> and <li> structure should be improved.   

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3089]
![image](https://user-images.githubusercontent.com/1005609/78629061-d7b38180-784a-11ea-8453-f92792151b9e.png)

**Actual**
Visually there is  one list with 2 list items but programmatically it is coded as 2 lists with 1 list item

**Expected**
`The programmatic structure should be <ul> <li>  <li>  </ul>    (instead of <ul> <li>  </ul>     <ul>  <li>  </ul>)`

**WCAG Violation**
Info and Relationships 1.3.1
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

**Comment by corinagum** [comment]
These are accessibility issues of v3 of Web Chat, which has been deprecated and we no longer support unless there are security risks. For accessible Web Chat, please [migrate to v4](https://github.com/Microsoft/BotFramework-WebChat#migrating-from-web-chat-v3-to-v4).

**Comment by ericmaino** [comment]
Confirmed this has been address in v4
